blog/fine-grained-news-2024-11.md

```diff
@@ -0,0 +1,107 @@
+- [Justin Cohen](https://www.linkedin.com/in/justincoh/) demonstrating the new Batch Check functionality.
+
+**Join us Thursday, December 12, 2024** for our next meeting. Can't make it? Catch up on our [latest recording](https://youtu.be/4MGF4rTzhbA?si=iGcoZTw8T99E0LKs) or browse previous sessions on our [YouTube channel](https://www.youtube.com/@OpenFGA).
+
+### **Blogs and Videos for AuthZ Fans:**  
+ - **Granting TTL based permissions in OpenFGA:** Implement TTL-based permissions in OpenFGA for time-limited access control. [Read more on Medium...](https://medium.com/@shruti1810/granting-ttl-based-permissions-in-openfga-2ed2073931c3)
+
+- **Overcoming Security Challenges in Protecting Shared Generative AI Environments:** Explore solutions for ensuring secure, scalable, and efficient multi-tenancy in generative AI environments. [Read more on Medium...](https://towardsdatascience.com/overcoming-security-challenges-in-protecting-shared-generative-ai-environments-1ffb27da1bde)
+
+- **Fine-Grained Authorization for Backstage using OpenFGA:** Learn how OpenFGA enables dynamic fine-grained authorization in Backstage through ReBAC models and seamless policy updates. [See the webinar on YouTube...](https://www.youtube.com/watch?v=wWFbLPvwOyQ)
+
+## **New Adopters**
+
+- Are you using OpenFGA in production? Join our growing community of adopters! Add your company to our [ADOPTERS.md](https://github.com/openfga/community/blob/main/ADOPTERS.md#companiesprojects-using-openfga-in-production) file with a quick PR.
+
+- Do you offer OpenFGA implementation services? Get listed in our [Implementation Services](https://github.com/openfga/community/blob/main/ADOPTERS.md#companies-offering-openfga-implementation-services) directory. Note: Listings are community-contributed and not officially endorsed by the OpenFGA project.
+
+## **Announcements**
+
+### **OpenFGA Rankked #5 in CNCF Project Contributions!**
+Thanks to our amazing community, OpenFGA soared to become the 5th most active CNCF project during Hacktoberfest in October! Your contributions made this possible, and hope to continue the engagement!
+
+Ready to join our community of contributors? We have opportunities for every skill level:
+- Start with our [Good First Issues](https://github.com/openfga/openfga/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for beginner-friendly tasks.
+- Take on more complex challenges in our [Issue queue](https://github.com/openfga/openfga/issues).
+- Follow our [Contribution Guide](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md) to get started.
+
+![CNCF projects ranked by commits during Hacktoberfest](../static/img/blog/fgn-2024-11-open-fga-ranks-5th.jpeg)
+
+*CNCF Projects Ranked by Commits during Hacktoberfest*
+
+### **Follow OpenFGA on LinkedIn**
+Connect with a growing community of fine-grained authorization enthusiasts and expand your professional network by following our new <!-- markdown-link-check-disable -->
```

Comment:
Unrelated to this post, can we add the LinkedIn page to the community page in docs? https://openfga.dev/docs/community

Reply:
Added LinkedIn page to Community page. PR: https://github.com/openfga/openfga.dev/pull/912